delfin/task_manager/tasks/resources.py

```diff
@@ -741,6 +815,12 @@ def sync(self):
             # Build relation between host grp and host to be handled here.
             storage_host_groups = self.driver_api \
                 .list_storage_host_groups(self.context, self.storage_id)
+            if storage_host_groups:
```

Comment:
Whenever we build group relations for any sync cycle, old relations to be flushed first.

Reply:
Already doing it, line no: 89,115,136

Reply:
yes, fine, missed it